lighthouse-core/config/seo.js

```diff
@@ -7,6 +7,15 @@
 
 module.exports = {
   extends: 'lighthouse:default',
+  passes: [{
+    passName: 'seoPass',
```

Comment:
patrick says you can just `passName: 'defaultPass'` and get this gather merged into that one.

so no need for an extra pass